activesupport/lib/active_support/callbacks.rb

```diff
@@ -92,14 +92,15 @@ module Callbacks
     # callback can be as noisy as it likes -- but when control has passed
     # smoothly through and into the supplied block, we want as little evidence
     # as possible that we were here.
-    def run_callbacks(kind)
+    def run_callbacks(kind, type = [:before, :around, :after])
```

Comment:
It would be best to constantize and freeze that default value to save an allocation per call.